packages/e2e-tests/specs/image-size.test.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * External dependencies
+ */
+import path from 'path';
+import fs from 'fs';
+import os from 'os';
+import uuid from 'uuid/v4';
+
+/**
+ * WordPress dependencies
+ */
+import {
+	createNewPost,
+	activatePlugin,
+	insertBlock,
+} from '@wordpress/e2e-test-utils';
+
+describe( 'changing image size', () => {
+	beforeEach( async () => {
+		await activatePlugin( 'gutenberg-test-image-size' );
+		await createNewPost();
+	} );
+
+	it( 'should insert and change my image size', async () => {
+		const expectedImageWidth = 499;
+
+		// Create a paragraph.
+		await insertBlock( 'Image' );
+		await page.click( '.editor-media-placeholder__media-library-button' );
+
+		// Wait for media modal to appear and upload image.
+		await page.waitForSelector( '.media-modal input[type=file]' );
+		const inputElement = await page.$( '.media-modal input[type=file]' );
+		const testImagePath = path.join( __dirname, '..', 'assets', '1024x768_e2e_test_image_size.jpg' );
+		const filename = uuid();
+		const tmpFileName = path.join( os.tmpdir(), filename + '.jpg' );
+		fs.copyFileSync( testImagePath, tmpFileName );
+		await inputElement.uploadFile( tmpFileName );
```

Comment:
Could you explain why you cannot use the file from `testFilePath` directly, to avoid needing to copy the file to a temporary directory?

Reply:
Well, I took the example provided in the issue, but you're right!
No need to copy the file, I'll test without it, and remove this part if it works great. 👍 

Thanks!

Reply:
I see the same pattern used for other files. It's probably fine to leave it as is.